### PR TITLE
Fixup location of assets for emails.

### PR DESCRIPTION
**Pull request by easternbloc**
Ensure we use the correct crest in our emails!

### CODE REVIEW

**Comment by easternbloc** [comment]
@JoeChapman @vaijab is it ok to use this url for the email assets?

@vaijab these assets wont resolve at present as we have an issue on the env.

Obvs this url (host) will change for go live...

File: services/email/templates/caseworker/html/delivery.mus [hunk 1]
```diff
@@ -34,9 +34,9 @@
...
                 <table width="100%" cellpadding="0" cellspacing="0" border="0" bgcolor="#FFFFFF">
                   <tr>
-                    <td width="6" style="border-left: 2px solid #009390;"><img src="http://govuk-prototyping.herokuapp.com/hmrc-email/spacer.gif" alt="" width="6px" height="26px" style="display: block;" /></td>
-                    <td width="26"><img src="http://govuk-prototyping.herokuapp.com/hmrc-email/hmrc_crest_18px.png" alt="HMRC Logo" style="width: 26px; height: 26px; display:block;"></td>
-                    <td width="7"><img src="http://govuk-prototyping.herokuapp.com/hmrc-email/spacer.gif" alt="" width="7px" height="26px" style="display: block;" /></td>
+                    <td width="6" style="border-left: 2px solid #009390;"><img src="https://brp-dev.notprod.homeoffice.gov.uk/public/images/email/spacer.gif" alt="" width="6px" height="26px" style="display: block;" /></td>
+                    <td width="26"><img src="https://brp-dev.notprod.homeoffice.gov.uk/public/images/email/ho_crest_27px.png" alt="HMRC Logo" style="width: 26px; height: 26px; display:block;"></td>
```
Comment (vaijab) [inline services/email/templates/caseworker/html/delivery.mus]: How is this going to work in a different environment?

**Comment by vaijab** [comment]
We should use a CDN for that.

**Comment by easternbloc** [comment]
> We should use a CDN for that.

^^^^ that!

**Comment by JoeChapman** [comment]
How about using data uri's for images? Then you wont have to worry about locations

**Comment by easternbloc** [comment]
> How about using data uri's for images? Then you wont have to worry about locations

Doing that now

**Comment by JoeChapman** [comment]
:octopus:

**Pull request merged**